src/main/resources/db/postgres/V1_89__vaas.sql

```diff
@@ -0,0 +1,12 @@
+CREATE TABLE VAAS
```

Comment:
kanskje et annet navn.. noe mer liknende topic-navnet?

Reply:
I don't think that it should be exactly like topic name. Maybe just `handelser`?